tests/suites/expose_ec2/expose_app.sh

```diff
@@ -109,7 +109,7 @@ assert_export_bundle_output_includes_exposed_endpoints() {
 
     got=$(juju export-bundle)
     exp=$(cat <<-EOF
-series: focal
+series: %s
```

Comment:
Why not grab the series before this line and expand it here instead of doing a hack with printf?